src/AutoRest.CSharp/Common/AutoRest/Plugins/CSharpProj.cs

```diff
@@ -114,7 +109,7 @@ public async Task<bool> Execute(IPluginCommunication autoRest)
 
                 if (configuration.AzureArm)
                 {
-                    csProjContent = string.Format(_csProjContent, additionalContent, _armCoreCsProjContent);
+                    csProjContent = string.Format(_csProjContent, additionalContent, _coreCsProjContent);
                 }
                 else
                 {
```

Comment:
We don't need this if, else condition, now that we are using the same core version. Code under the if, else is exactly the same, we can remove the if, else now.